lib/comment.js

```diff
@@ -28,27 +28,35 @@ module.exports = function(number, metadata) {
         });
     }
 
-    // if there are collaborators, we ask them for a review
-    if (metadata.reviewersExcludingAuthor.length > 0 || metadata.isRoot) {
-        if (metadata.missingReviewers.length > 0) {
-            return  github.post('/repos/:owner/:repo/pulls/:number/requested_reviewers', {
-                reviewers: metadata.missingReviewers
-            }, { number: number }).then(function() {
-                if (!metadata.isMergeable) {
-                    return labelAndComment(number, "status:needs-collaborator", "No reviewer on this pull request has write-access to the repository.");
-                }
-            });
-        } else if (metadata.isMergeable) {
-            return promise();
-        } else {
-            return labelAndComment(number, "status:needs-collaborator", "No reviewer on this pull request has write-access to the repository.");
+    promise().then(function() {
```

Comment:
We'll need to return this value so [the caller](https://github.com/web-platform-tests/wpt-pr-bot/blob/e571d166c6fcbf8cac635038b281582d2455481e/index.js#L108) can track the outcome.

Reply:
Done.